src/node_contextify.cc

```diff
@@ -658,6 +660,51 @@ class ContextifyScript : public BaseObject {
   }
 
 
+  static Local<Integer> GetLineOffsetArg(
+                                      const FunctionCallbackInfo<Value>& args,
+                                      const int i) {
+    Local<Integer> defaultLineOffset = Integer::New(args.GetIsolate(), 0);
+
+    if (args[i]->IsUndefined()) {
+      return defaultLineOffset;
+    }
+    if (args[i]->IsInt32()) {
+      return args[i].As<Integer>();
+    }
+    if (!args[i]->IsObject()) {
```

Comment:
I think this is the only check you actually need here. The `IsUndefined()` and `IsInt32()` checks above can go. Same comment applies for your `GetColumnOffsetArg()` function.